Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ghcr.io/shyim/wolfi-php/frankenphp:8.3 AS base
+RUN apk add --no-cache \
+    php-frankenphp-8.3-opcache \
+    php-frankenphp-8.3-mbstring \
+    php-frankenphp-8.3-intl \
+    php-frankenphp-8.3-dom \
+    php-frankenphp-8.3-curl
+
+FROM base AS prod
```

Comment:
It's not usable aside from the rest stuff like Makefile or docker-compose* files

Reply:
Where should it be used?